LDAR_Sim/src/initialization/preseed.py

```diff
@@ -52,29 +55,33 @@ def get_seed_timeseries(gen_dir) -> list[int]:
     return seed_ts
 
 
-def gen_seed_emis(n_sim: int, gen_dir):
+def gen_seed_emis(n_sim: int, gen_dir) -> tuple[list[int], bool]:
     preseed_val: list[int] = []
     preseed_loc = gen_dir / Generator_Files.EMISSION_PRESEED_FILE
+    force_remake = False
     if not os.path.exists(gen_dir):
+        force_remake = True
         os.mkdir(gen_dir)
 
     if os.path.isfile(preseed_loc):
         preseed_val = get_emis_seed(gen_dir)
         if len(preseed_val) < n_sim:
-            print("Generating additional Random Seed values for emissions...")
+            force_remake = True
```

Comment:
Wondering if this should be true here - This is for the case where everything is the same, but now there are more simulations.

Reply:
You're correct. Removed.